sources/reference/resource-state.md

```diff
@@ -0,0 +1,38 @@
+page_main_title: state
+main_section: Reference
+sub_section: Resources
+
+# state
+A `state` resource type is used to create a central state that can be used across the entire pipelines. This resource can be used as an IN input and OUT output for [any job](jobs-overview/). The timezone used for triggering jobs is UTC.
+
+You can create this resource by adding it to `shippable.resources.yml`
+
+```
+resources:
+  - name: res-state
+    type: state
+```
+
+* `name` should be an easy to remember text string. This will appear in the visualization of this resource in the SPOG view. It is also used to refer to this resource in the `shippable.jobs.yml`. If you have spaces in your name, you'll need to surround the value with quotes, however, as a best practice we recommend not including spaces in your names.
+
+* `type` is always set to 'state'.
+
+You can use the time resource as an input and output for any job you want to trigger at that interval:
+
+```
+jobs:
+  - name: tfDeploy
+    type: runSh
+    steps:
+      - IN: integration-aws
+      - IN: repo-tfScripts
+      - IN: res-state
+      - TASK:
+        - script: ./doSomething.sh
+      - OUT: res-state
+
+```
+
+* Updating the resource version for `type: state` will not trigger subsequent jobs.
+
+* Cyclic dependency will be created when we input and output the `state` resource to a job.
```

Comment:
what does this line mean to a user? are you saying they shouldn't do that? The UI handles the cycle, and it doesn't impact how a user uses it, so i think we should not say this.